Server/src/index.ts

```diff
@@ -302,13 +303,28 @@ export class CitrineOSServer {
   }
 
   private initLogger() {
-    return new Logger<ILogObj>({
+    const logger = new Logger<ILogObj>({
       name: 'CitrineOS Logger',
       minLevel: systemConfig.logLevel,
       hideLogPositionForProduction: systemConfig.env === 'production',
       // Disable colors for cloud deployment as some cloud logging environments such as cloudwatch can not interpret colors
       stylePrettyLogs: process.env.DEPLOYMENT_TARGET !== 'cloud',
     });
+    const logstashHost = 'logstash'; // WARNING: This will only work within docker
```

Comment:
This seems like a serious issue, right? what happens when you start locally?

Reply:
then it just logs to the console.. because this is just an additional transport. You do make a good point though, this was kind of a quick intro PR to get thoughts from the team on where we want to take it - in this case, if you run locally, your local config would specify where you are running logstash or exclude it entirely.. so if you have logstash running locally, you can use `localhost` here